test/extensions/filters/network/redis_proxy/redis_proxy_integration_test.cc

```diff
@@ -154,7 +155,8 @@ const std::string CONFIG_WITH_ROUTES_BASE = R"EOF(
           stat_prefix: redis_stats
           settings:
             op_timeout: 5s
-)EOF";
+)EOF",
+                                                        TestEnvironment::nullDevicePath());
```

Comment:
This one is too far? 

Format check failed, try apply following patch to fix:
```diff
diff --git a/test/extensions/filters/network/redis_proxy/redis_proxy_integration_test.cc b/test/extensions/filters/network/redis_proxy/redis_proxy_integration_test.cc
index b67cab241..928758cd7 100644
--- a/test/extensions/filters/network/redis_proxy/redis_proxy_integration_test.cc
+++ b/test/extensions/filters/network/redis_proxy/redis_proxy_integration_test.cc
@@ -60,7 +60,7 @@ static_resources:
           settings:
             op_timeout: 5s
 )EOF",
-                                                        TestEnvironment::nullDevicePath());
+                                       TestEnvironment::nullDevicePath());

// This is a configuration with command stats enabled.
const std::string CONFIG_WITH_COMMAND_STATS = CONFIG + R"EOF(
```

Reply:
I am getting really confused... I am pretty sure I had it like this in the previous commit. Look at the diff here 7cedce818faeb27c096d44cb5eb96999a8a3d715